fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/door_animations.xml

```diff
@@ -11,7 +11,9 @@
 			</UseTemplate>
             <UseTemplate Name="ASOBO_GT_Interaction_LeftSingle_Leave_Code">
                 <LEFT_SINGLE_CODE>
-                    #TOGGLE_ID# (&gt;K:TOGGLE_AIRCRAFT_EXIT)
+                    (L:A32NX_PRESS_MAN_CABIN_DELTA_PRESSURE, psi) 0.8 &lt; (A:INTERACTIVE POINT OPEN:#ID#, Percent) 0 &gt; || if{
```

Comment:
In practice, this should never happen that the cabin is pressurized and the door is open, however this condition allows for correcting possible bugs where if the plane spawns with the door open (and cabin is pressurized), that the door can still be closed.

Reply:
When the pressurization simulation reads all door states, this will never happen as you would get cabin decompression if a door is open with delta P > 0. But since that's still not done, this looks good to me.